src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+:root {
+  --active-color: #00acdc;
+}
+
+.body {
```

Comment:
It's better to use `body` instead of `.body` for the body tag styles. This way, you don't need to add a class 'body' to your HTML body tag 🌐